src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -26,42 +25,75 @@ public class StationAcceptanceTest {
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
-        // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
-
-        // then
+        ExtractableResponse<Response> response = createSubwayStation("신림역");
         assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
-
-        // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 조회한다")
+    @Test
+    void getStation() {
+
+        //given
+        final String first_station = "신림역";
+        final String second_station = "봉천역";
+        createSubwayStation(first_station);
+        createSubwayStation(second_station);
```

Comment:
```suggestion
        createSubwayStation("신림역");
        createSubwayStation("봉천역");
```

변수 선언 없이 바로 넣어줘도 괜찮지 않을까요?